commit b487bbfeb2ecdf766b634ac907281fbd8559a2bd (refs/changes/29/429/1)
Author: Orlando Vazquez <ovazquez@gmail.com>
Date:   2016-09-07T15:34:41-07:00 (3 years, 1 month ago)
    
    CNAPI-669 CNAPI waitlist paging test takes a very long time to complete

diff --git a/test/waitlist/test-waitlist.js b/test/waitlist/test-waitlist.js
index d29684a..6b193b5 100644
--- a/test/waitlist/test-waitlist.js
+++ b/test/waitlist/test-waitlist.js
@@ -438,10 +438,10 @@ function testLimitOffsetValidation(test) {
  */
 
 function testFetchTicketsWithPaging(test) {
-    test.expect(6641);
+    test.expect(638);
 
-    var count = 1100;
-    var limit = 100;
+    var count = 100;
+    var limit = 10;
     var offset = 0;
 
     var ticketUuids;
